pkg/email/send_email.go

```diff
@@ -38,10 +46,13 @@ func NewEmailer(p Params) *Emailer {
 		Params:  p,
 		Dialer:  dialer,
 		Limiter: limiter,
-	}
+	}, nil
 }
 
 func (e *Emailer) SendEmail(receivers []string, subject string, body string, attachmentFilePath []string) error {
+	if e.Params.SMTPHost == "" {
+		return ErrNoSMTPHostConfigured
+	}
```

Comment:
Same verification should be done for 'Sender' and 'receivers'

Reply:
Done